modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -988,6 +988,11 @@ enum EditType {
   EditType Updated
   """
   UPDATED
+
+  """
+  EditType Deleted
+  """
+  DELETED
```

Comment:
Seems like someone might be forgiven for thinking that they'd get a `DELETED` event when setting the existence flag to `DELETED` but they'd be wrong?

Reply:
Yes, they'd be wrong

Reply:
Do you think another word would be more appropriate?